.travis.yml

```diff
@@ -56,6 +57,8 @@ matrix:
           env: PYTHON_VERSION=3.3 SETUP_CMD='test --open-files'
         - os: linux
           env: PYTHON_VERSION=3.4 SETUP_CMD='test --open-files'
+        - os: linux
+          env: PYTHON_VERSION=3.5 SETUP_CMD='test --open-files'
```

Comment:
Yeah -- I just wanted to test it first, because the failures on 3.5 have been spurious, and I was hoping I could knock them out altogether.